### PR TITLE
staff admin page fixes

### PR DESCRIPTION
**Pull request by achen2401**
this should address issues raised by story:
https://www.pivotaltracker.com/story/show/143217817
- staff list for admin staff user will now exclude user(s) of admin or admin_staff role from the same org as the user
- allow staff admin user to create account of staff user of all orgs (not just leave orgs) of the user
- allow staff to send registration email for newly created user

### CODE REVIEW

File: portal/templates/profile_macros.html [hunk 1]
```diff
@@ -366,7 +368,87 @@ <h4 class="text-success registration-label">registered</h4>
...
+                body += "<p>This email was sent to you from " + clinicName + ". This is an invitation to complete your registration.</p>";
+                body += "Verify your account now by clicking on the link below to visit TrueNTH:";
+                body += "<br/><br/>";
+                body += "<a href='" + return_url + "'>Complete Registration</a>";
+                subject = "Registration invite from " + clinicName;
+            };
+            if (body != '') {
+                $.ajax ({
+                    type: 'POST',
+                    url: '/invite',
```
Comment (ivan-c) [inline portal/templates/profile_macros.html]: Might be good to use `url_for()` here...

File: portal/templates/staff_profile_create.html [hunk 2]
```diff
@@ -197,7 +197,7 @@ <h4 class="tnth-headline left-indent-top">{{ _("New Staff") }}</h4>
                 }).get();
 
                 _accountArray['organizations'] = orgIDs;
-                _accountArray['roles'] =  [{'name': 'staff'}];
+                _accountArray['roles'] =  [{'name': 'staff'}, {'name': 'write_only'}];
```
Comment (ivan-c) [inline portal/templates/staff_profile_create.html]: Might be a good idea to use the strings from the enumerated roles if you can, eg `ROLE.WRITE_ONLY`. Not a big deal but would make refactoring much easier if we decided to rename them...

File: portal/templates/profile_macros.html [hunk 3]
```diff
@@ -366,7 +368,87 @@ <h4 class="text-success registration-label">registered</h4>
...
+                        };
+                    });
+                };
+                var cn = parentName ? parentName: "your clinic";
+                return cn;
+            })();
+            return_url = getAccessUrl();
+            if (hasValue(return_url)) {
+                body = "<p>Hello,</p>";
+                body += "<p>This email was sent to you from " + clinicName + ". This is an invitation to complete your registration.</p>";
```
Comment (achen2401) [inline portal/templates/profile_macros.html]: hmm... clinicName is generated dynamically from client side here, not sure if I can apply gettext in this context

**Comment by pbugni** [comment]
Hi @achen2401 - we've changed several of these files since this PR was generated.  Would you mind rebasing?

**Comment by achen2401** [comment]
@pbugni Paul, thank you, conflicts should be resolved in this PR.

**Pull request merged**